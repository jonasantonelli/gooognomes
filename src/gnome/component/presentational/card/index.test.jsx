/**
 * @jest-environment jsdom
 */
import React from 'react';
import { shallow } from 'enzyme';
import Card from './index.jsx';

describe('Card Component', () => {

  let wrapper;

  const props = {
    id: 1,
    thumbnail: 'http://www.publicdomainpictures.net/pictures/10000/nahled/1-1275919724d1Oh.jpg',
    name: 'Test',
    age: 100
  }

  beforeEach(() => {
    wrapper = shallow(<Card {...props} />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should contain title class', () => {
    expect(wrapper.exists('.card')).toEqual(true);
  });

});
