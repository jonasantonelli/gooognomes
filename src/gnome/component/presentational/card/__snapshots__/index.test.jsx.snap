// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component should match snapshot 1`] = `
<NavLink
  className="card"
  to="/details/1"
>
  <img
    alt="Test"
    src="http://www.publicdomainpictures.net/pictures/10000/nahled/1-1275919724d1Oh.jpg"
  />
  <div
    className="card__info"
  >
    <div
      className="card__title"
    >
      Test
    </div>
    <div
      className="card__age"
    >
      Age: 
      100
    </div>
  </div>
</NavLink>
`;
